receiver/mysqlreceiver/metadata.yaml

```diff
@@ -488,6 +488,15 @@ metrics:
       input_type: string
       monotonic: true
       aggregation_temporality: cumulative
+  mysql.max_used_connections:
+    enabled: true
```

Comment:
you cannot add this new metric as enabled by default unless it's a breaking change. See https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/scraping-receivers.md#changing-the-emitted-metrics